tests/perforce_utils.py

```diff
@@ -108,6 +108,14 @@ def create_new_p4_container(request, client, name):
         request.raiseerror("Timeout on waiting perforce server to start.")
 
 
+class PerfoceDockerContainer:
```

Comment:
You should be able to use this class as annotation in the usage of the fixture, like follows:
```
def perforce_connection(request, docker_perforce: PerfoceDockerContainer):
```